resources/store/index.js

```diff
@@ -197,6 +197,7 @@ module.exports = new Vuex.Store( {
 		 */
 		setTab: function ( state, tabName ) {
 			ensureTabExists( state, tabName );
+			window.history.replaceState( null, null, '#' + tabName );
```

Comment:
it felt a little weird putting this here since it's not a state mutation, but I thought it belonged after the ensureTabExists check.

Reply:
Yeah, we may want to re-think this. Vuex in our case should know nothing about the URL and should not care; URL changes should trigger changes in state just like other user interactions in components.

We could potentially use the error behavior of `ensureTabExists` to handle a fragment corresponding to a non-existing tab, but in that case I think a try/catch block in an appropriate component would be more appropriate. This would likely belong in a watcher and/or created/mounted hook for the App component; the catch behavior would just tell Vuex to switch to the popular tab.

Reply:
I removed this from the store and am instead applying it in a few places where we're changing tabs and need to update the URL fragment to match